drivers/overlay/overlay.go

```diff
@@ -431,6 +439,25 @@ func (d *Driver) Get(id string, mountLabel string) (s string, err error) {
 		return "", err
 	}
 
+	newlowers := ""
+	for _, lower := range strings.Split(string(lowers), ":") {
+		newpath := path.Join(d.home, lower)
+		if _, err := os.Stat(newpath); err != nil {
+			for _, p := range d.AdditionalImageStores() {
+				lower := path.Join(p, d.name, lower)
+				_, err = os.Stat(lower)
+				if err == nil {
+					break
+				}
+			}
```

Comment:
Is this `for` loop doing what it's supposed to?  Neither `p` defined on line 446 nor the new `lower` defined on line 447 seem to be affecting anything outside of the loop's scope.